scripts/webpack/webpack-resources.js

```diff
@@ -350,15 +349,17 @@ module.exports = {
         // Use the aliases for react-examples since the examples and demo may depend on some things
         // that the package itself doesn't (and it will include the aliases for all the package's deps)
         alias: getResolveAlias(false /*useLib*/, reactExamples),
+        fallback: {
+          path: require.resolve('path-browserify'),
+        },
       },
     });
   },
 };
 
 function getPlugins(bundleName, isProduction, profile) {
   const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
-
-  const plugins = [];
+  const plugins = [new webpack.DefinePlugin(getDefaultEnvironmentVars(isProduction))];
```

Comment:
Is this because DefinePlugin used to run by default but now it doesn't?

Reply:
node environment variables used to be auto replaced by webpack, and now they must be explicit.